packages/components/psammead-media-player/src/Guidance/index.test.jsx

```diff
@@ -8,6 +8,7 @@ describe('Media Player: Guidance', () => {
     <Guidance
       guidanceMessage="Guidance: Contains strong language with adult humor"
       service="news"
+      noJsMessage="no js"
```

Comment:
I'm concerned why this shows as not been added but they're added on latest [here](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-media-player/src/Guidance/index.test.jsx#L19)